src/monodroid/jni/debug.cc

```diff
@@ -213,13 +213,8 @@ Debug::start_debugging_and_profiling ()
 		total_time.mark_start ();
 
 	char *connect_args;
```

Comment:
For absolute correctness, please set `connect_args` to `nullptr` here should the call to `androidSystem.monodroid_get_system_property` ever fail to set the pointer to a meaningful value (it's OK to pass a `nullptr` to `delete`, not so much a pointer with any other value that points to a random location)